tensorrt_llm/llmapi/llm.py

```diff
@@ -661,6 +662,11 @@ def _build_model(self):
                 backend=tllm.GuidedDecodingConfig.GuidedDecodingBackend.
                 XGRAMMAR,
                 **_xgrammar_tokenizer_info(self.tokenizer))
+        elif self.args.guided_decoding_backend == 'llguidance':
+            executor_config.guided_decoding_config = tllm.GuidedDecodingConfig(
```

Comment:
This is caused by rebase conflict:

```suggestion
            self._executor_config.guided_decoding_config = tllm.GuidedDecodingConfig(
```